go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pingcap/ng-monitoring
 
-go 1.24.6
+go 1.25.0
```

Comment:
Why do we want to update to this new version? Any considerations? I see the tidb repo is still an older version

Reply:
Aligned to tidb, which will update the version soon.